add filter for projects items

IgorZiegler opened the pull request.

Commit:
add filter for projects items

knutole commented on js/src/chrome/users/chrome.users.js:
```diff
@@ -489,6 +499,7 @@ Wu.Chrome.Users = Wu.Chrome.extend({
 
 	_createInviteInput : function (options) {
 
+		var me = this;
```
@IgorZiegler  please don't use `me`... `this` is perfectly fine, and it's what's being used everywhere else in the code. It's confusing to have `me` as context. thanks.

IgorZiegler replied:
@knutole Okay i will fixed it. It is just my habit=) Like closure=)

Commit:
refactoring code style

knutole commented:
### bug:

When selecting a project (for example `NVE`) and adding to list, it works fine. However - if you exit the fullscreen and open it again, the `NVE` project is no longer available. Probably just a matter of resetting properly when opening fullscreen again.
![screen shot 2016-02-16 at 4 03 23 pm](https://cloud.githubusercontent.com/assets/2197944/13079932/dcd4fd00-d4c6-11e5-8227-71aeaefe7afa.png)
![screen shot 2016-02-16 at 4 03 30 pm](https://cloud.githubusercontent.com/assets/2197944/13079935/df348354-d4c6-11e5-8800-7fb52f2c78d3.png)
![screen shot 2016-02-16 at 4 03 36 pm](https://cloud.githubusercontent.com/assets/2197944/13079938/e283e720-d4c6-11e5-8d3a-5a64bbe92d32.png)

Commit:
add close full screen event

Commit:
fixed bug with get shareable link

Commit:
change fullscreen event handler

knutole committed:
Small fixes:
- fixed context issue for this, using .call(this)
- renamed to _onCloseFullscreen
